src/annotator/config/is-browser-extension.js

```diff
@@ -6,6 +6,6 @@
  *   website.
  *
  */
-export default function isBrowserExtension(app) {
+export function isBrowserExtension(app) {
```

Comment:
This would be a good time to rename `app` to something better (perhaps just `url`?) and add a `@param` entry to document its type.